backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/picking/job/model/PickingJobStepPickedToHU.java

```diff
@@ -18,5 +19,6 @@ public class PickingJobStepPickedToHU
 	@NonNull HUInfo actualPickedHU;
 	@NonNull Quantity qtyPicked;
 	@Nullable Quantity catchWeight;
+	@Nullable Instant createdAt;
```

Comment:
why nullable?
can't we set SystemTime.asInstant() in case it's not set?

Reply:
yeah, we can :+